src/scripts/qa.ts

```diff
@@ -1,23 +1,13 @@
-import {
-  Chain,
-  ChainId,
-  ChainType,
-  HashflowApi,
-  validateChain,
-} from '@hashflow/taker-js';
-import { PriceLevel } from '@hashflow/taker-js/dist/types/common';
+import {Chain, ChainId, ChainType, HashflowApi, validateChain,} from '@hashflow/taker-js';
+import {PriceLevel} from '@hashflow/taker-js/dist/types/common';
 import BigNumber from 'bignumber.js';
-import { getSecretValue } from 'helpers/secrets';
+import {computeLevelsQuote} from 'helpers/levels';
+import {getSecretValue} from 'helpers/secrets';
+import {convertFromDecimals, convertToDecimals} from 'helpers/token';
+import {Environment, Token} from 'helpers/types';
+import {validateEnvironment, validateMakerName,} from 'helpers/validation';
 import yargs from 'yargs/yargs';
-
-import { computeLevelsQuote } from '../helpers/levels';
-import { convertFromDecimals, convertToDecimals } from '../helpers/token';
-import { Environment, Token } from '../helpers/types';
-import {
-  validateAddress,
-  validateEnvironment,
-  validateMakerName,
-} from '../helpers/validation';
+import {validateEvmAddress, validateSolanaAddress} from '../../../taker-js/src/helpers/validation';
```

Comment:
we shouldn't need to go up that many directories for import, I guess there is something wrong with IDE

Reply:
Yeah it's just because I'm importing from my local copy of taker-js for local testing. Once the taker-js PR lands and is published this can be fixed to have a correct import.

Reply:
I've had spotty success with `yarn link` -- i figured this worked for local testing, so i didn't want to spend extra time on getting it to work as i knew i'd fix it once the taker-js Pr lands.